### PR TITLE
Fix background color with GTK3

### PR DESCRIPTION
**Pull request by livanh**
I tried building for gtk3, and the background color was wrong (it was the same as the toolbars - see attached picture, left side). This is how I fixed it (result on right side). I can make changes if required for merging.

image

### CODE REVIEW

**Comment by wb9688** [comment]
I also noticed this issue, though it only happens when you have a single tab open.

File: src/gtk/exo/exo-icon-view.c [hunk 1]
```diff
@@ -1878,6 +1878,16 @@ exo_icon_view_expose_event (GtkWidget      *widget,
 
   if (!gtk_cairo_should_draw_window (cr, priv->bin_window))
     return FALSE;
+
+  /* draw a background according to the css theme */
+  style = gtk_widget_get_style_context (widget);
```
Comment (LStranger) [inline src/gtk/exo/exo-icon-view.c:1883]: I suppose it should be not retrieved again at line 1993 with this addition, right?

Comment (livanh) [reply src/gtk/exo/exo-icon-view.c:1883]: Correct, I just tried and it works the same. PR updated

File: src/gtk/exo/exo-icon-view.c [hunk 2]
```diff
@@ -1979,9 +1989,7 @@ exo_icon_view_expose_event (GtkWidget      *widget,
     }
 
   if (G_UNLIKELY (dest_item != NULL || priv->doing_rubberband))
-#if GTK_CHECK_VERSION(3, 0, 0)
-      style = gtk_widget_get_style_context (widget);
-#else
+#if !GTK_CHECK_VERSION(3, 0, 0)
```
Comment (LStranger) [inline src/gtk/exo/exo-icon-view.c:1992]: This line should be moved above `if` to not leave it alone.

Comment (livanh) [reply src/gtk/exo/exo-icon-view.c:1991]: You're right. I should look twice before committing. Fixed

**Comment by ib** [comment]
@livanh, are you okay with the merge and the resulting commit?

**Comment by ib** [comment]
Thank you very much for your contribution.

**Pull request merged**